op-service/queue/queue.go

```diff
@@ -0,0 +1,25 @@
+package queue
+
+type Queue[T any] []T
```

Comment:
A few unit tests of this generic impl would be nice, e.g. on a `Queue[int]`.

Reply:
[f3af344](https://github.com/ethereum-optimism/optimism/pull/12180/commits/f3af344764161e74a58564f98794e4c748b48772)